inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd

```diff
@@ -1130,38 +1300,39 @@ it will then pull the age group with the highest proportion of pregnant women.
 
 pregnant_women_in_data <- study_data_cleaned %>% 
```

Comment:
here too - tabyl simpler for the purpose (and table not displayed anyway)

Reply:
Let's go with simpler. 
Do tabyl tables knit well to doc?